seccomp.go

```diff
@@ -117,6 +117,10 @@ const (
 	ArchS390 ScmpArch = iota
 	// ArchS390X represents 64-bit System z/390 syscalls
 	ArchS390X ScmpArch = iota
+	// ArchPARISC represents 32-bit PA-RISC
+	ArchPARISC ScmpArch = iota
```

Comment:
Foolish question from a golang neophyte - would we be better off setting the first value (ArchInvalid in this case) equal to `iota` and leaving the rest blank.

Reply:
I *thought* the convention was to use `iota` for all the entries, and the current use seems to bear that out.  However, like you I am far from a golang expert.